rasa/cli/data.py

```diff
@@ -254,57 +262,26 @@ def _convert_to_yaml(args: argparse.Namespace, is_nlu: bool) -> None:
+    for root, _, files in os.walk(training_data, followlinks=True):
+        for f in sorted(files):
+            source_path = Path(os.path.join(root, f))
 
-        if MarkdownReader.is_markdown_nlu_file(source_path):
-            if not is_nlu:
+            if not is_valid_filetype(source_path):
                 continue
-            _write_nlu_yaml(source_path, output_path, source_path)
-            num_of_files_converted += 1
-        elif not is_nlu and MarkdownStoryReader.is_markdown_story_file(source_path):
-            _write_core_yaml(source_path, output_path, source_path)
-            num_of_files_converted += 1
-        else:
-            print_warning(f"Skipped file: '{source_path}'.")
 
-    print_info(f"Converted {num_of_files_converted} file(s), saved in '{output}'.")
-
-
-def _write_nlu_yaml(
-    training_data_path: Path, output_path: Path, source_path: Path
-) -> None:
-    reader = MarkdownReader()
-    writer = RasaYAMLWriter()
+            if MarkdownReader.is_markdown_nlu_file(source_path):
+                if not is_nlu:
+                    continue
+                write_nlu_yaml(source_path, output)
+                num_of_files_converted += 1
+            elif not is_nlu and NLGMarkdownReader.is_markdown_nlg_file(source_path):
+                write_nlg_yaml(source_path, output)
+                num_of_files_converted += 1
+            elif not is_nlu and MarkdownStoryReader.is_markdown_story_file(source_path):
+                write_core_yaml(source_path, output)
+                num_of_files_converted += 1
+
+            else:
+                print_warning(f"Skipped file: '{source_path}'.")
 
-    training_data = reader.read(training_data_path)
-    writer.dump(output_path, training_data)
-
-    print_success(f"Converted NLU file: '{source_path}' >> '{output_path}'.")
-
-
-def _write_core_yaml(
-    training_data_path: Path, output_path: Path, source_path: Path
-) -> None:
-    from rasa.core.training.story_reader.yaml_story_reader import KEY_ACTIVE_LOOP
-
-    reader = MarkdownStoryReader()
-    writer = YAMLStoryWriter()
-
-    loop = asyncio.get_event_loop()
-    steps = loop.run_until_complete(reader.read_from_file(training_data_path))
-
-    if YAMLStoryWriter.stories_contain_loops(steps):
-        print_warning(
-            f"Training data file '{source_path}' contains forms. "
-            f"Any 'form' events will be converted to '{KEY_ACTIVE_LOOP}' events. "
-            f"Please note that in order for these stories to work you still "
-            f"need the 'FormPolicy' to be active. However the 'FormPolicy' is "
-            f"deprecated, please consider switching to the new 'RulePolicy', "
-            f"for which you can find the documentation here: {DOCS_URL_RULES}."
-        )
-
-    writer.dump(output_path, steps)
-
-    print_success(f"Converted Core file: '{source_path}' >> '{output_path}'.")
+    print_info(f"Converted {num_of_files_converted} file(s), saved in '{output}'.")
```

Comment:
Can we print another message if `num_of_files_converted` is 0?

Reply:
good point!